### DISCUSSION TITLE
Hyprdots-ctl Updates:

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#690] [Show and tell]
I will use this Thread to show some progress on Hyprdots-ctl

``` Hyprdots show binds ``` waiting for implementation of #477 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/6a726a49-e8f2-461d-a964-5af825f61513)

``` Hyprdots show emoji ``` lightweight emoji selector that matches Hyprdots themes 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/adbf151f-71a0-4341-9a48-782bcb727468)

**Comment by Gl00ria** [comment]
@kRHYME7 hey, long time no see huh.

Ive seen your pull request & your comment on the older pull request regarding this, what you've done is lovely and I like it.
I did test & took notes regarding the one you opened a pull request for at the same day you submitted the pull request, but later I got swamped in a job overload so I couldn't reply back.

Anyway, I cannot find my notes at the moment **sorry**, but I remember that it was perfectly working. I will look into this one as soon as I find time to breathe lol.

BTW, i see you active everywhere (here & reddit) have a lil bit of a break hahah.  

Cheers for the time & effort you put here.

**Reply by kRHYME7** [reply]
> BTW, i see you active everywhere (here & reddit) have a lil bit of a break hahah.

🤣  I'll take that advice. I always bring my laptop anywhere, and every time I get to sit and got free time I always try to solve issues or whatever comes to mind.  I love learning new things, but I guess I'll take this as a cue to breathe 😄. Happy Holidays to you @Gl00ria.

**Reply by kRHYME7** [reply]
Hi @Gl00ria  already touched grass and guess what, parsing comments was just a one-liner. #665  Those 3 active PRs will be enough for this Year. Haha. Hoping for you to review if you soon have the luxury of time.  Thank you and have a Happy New Year!

**Comment by kRHYME7** [comment]
# Pastebin and Emoji selector
### Auto insert for
``` Hyprdots show pastebin ```  and ``` Hyprdots show emoji ```

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/7f64ee22-375b-46d3-90e8-2cbdba3564ab)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/db76236f-0dfb-4426-a160-fb792ab45200)

### multiselect is experimental do not paste large quantity in the terminal. 



藍朗☺️浪螺來狼樂蘿廊拉襤裸洛濫冷蠟掠略勞

**Comment by kRHYME7** [comment]
## Hyprdots show glyph

For anyone who wants to have glyphs on the go. ()

Glyphs are pre-fetch from https://www.nerdfonts.com/cheat-sheet

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9dd899a0-08aa-4f17-8e75-84804bbf2c43)



~~#TODO:  How to get the emoji from upstream Nerdfonts~~ Done

者
